tests/activities/models/test_post.py

```diff
@@ -112,3 +115,45 @@ def test_linkify_mentions_local(identity, remote_identity):
         local=True,
     )
     assert post.safe_content_local() == "<p>@test@example.com, welcome!</p>"
+
+
+async def stator_process_tasks(stator):
```

Comment:
I think I might just move this into the stator runner, we do need something like this!